db/init_db.js

```diff
@@ -1,31 +1,193 @@
 // code to build and initialize DB goes here
 const {
-  client
+  client,
   // other db methods 
 } = require('./index');
 
-async function buildTables() {
+const {createUser} = require('./db')
+const {createProduct} = require('./products')
+
+async function dropTables() {
+  client.connect();
+  console.log("Dropping Tables ...")
   try {
-    client.connect();
+    await client.query(`
+      DROP TABLE IF EXISTS order_products;
+      DROP TABLE IF EXISTS orders;
+      DROP TABLE IF EXISTS products;
+      DROP TABLE IF EXISTS users;
+    `);
+  } catch (error) {
+    throw error
+  }
+}
 
-    // drop tables in correct order
+async function buildTables() {
+  console.log("Building Tables ...")
+  try {
+    await client.query(`
+      CREATE TABLE users(
+        id SERIAL PRIMARY KEY,
+        "firstName" VARCHAR(255) NOT NULL,
+        "lastName" VARCHAR(255) NOT NULL,
+        email VARCHAR(255) UNIQUE NOT NULL,
+        imageURL VARCHAR(255) DEFAULT 'imageUrl',
```

Comment:
Should imageURL be in quotation marks since it has capital letters? Here and in products table below.

Reply:
I think quotes would be implemented for imageURL just in the creating tables. Looking at previous projects it wouldn't be necessary for where it is in the products table

Reply:
I think this has to do with preserving the capitalization, since PostgreSQL converts everything to lowercase if not quoted. So we will have to query using imagurl instead of imageURL: https://blog.xojo.com/2016/09/28/about-postgresql-case-sensitivity/

Reply:
OH! right right i just read through that. Makes sense. Ok i'll add quotes to that part then!